packages/client/css/helper-lines.css

```diff
@@ -16,8 +16,8 @@
 
 .helper-line {
     pointer-events: none;
-    stroke: red;
-    stroke-width: 1px;
+    stroke: #f7086c;
```

Comment:
Should we maybe use a less aggressive color than red for the helper lines?

Reply:
Do you maybe have a suggestion? I tried to choose a softer red, more pink-ish color to make it less look like an error. However, I couldn't quickly find a color that has a good contrast to most other colors and still look very visible on the blue-ish background.

Reply:
I pushed an update where I set the stroke width back to 1 and use the blue that we also use for the resize handles, please have a look at that.